CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+Issues and pull requests are very welcome for bug reports and feature requests and/or proposals.
+
+Please provide as much information as possible, describing the bug or feature in detail. Screenshots and videos are great.
+
+All code needs to include programmatic tests, and to have been tested manually as well.
+
+If you're interested in contributing to the project long term (every skill level welcome), let us know [over email](mailto:oli.azevedo.barnes@gmail.com)  and we'll onboard you through a call and adding you on our slack.
+
+Please note that for all communication and interactions we expect our [code of conduct](CODE_OF_CONDUCT.md) to be followed.
+
+## Documentation
+
+Documentation can be found at
+
+- The [liquidvoting.io](https://liquidvoting.io) website, edited on its [github page repo](https://github.com/liquidvotingio/liquidvotingio.github.io)
+- READMES for [the api](https://github.com/liquidvotingio/api) and [auth]([api](https://github.com/liquidvotingio/api) repos
+- Code comments
+- Tests
+
+Keeping docs up to date are part of daily work - so whenever we add a new feature, they are part of the delivery. Of course, stuff some times falls through the cracks, so whenever we notice some documentation is missing, we either add it or create an issue for it in case it's more involved.
```

Comment:
```suggestion
Keeping docs up to date are part of daily work: whenever we add a new feature, they are part of the delivery. Of course, stuff sometimes falls through the cracks but whenever we notice some documentation is missing, we either add it or create an issue for it in case it's more involved.
```